assets/css/frontend-form/layout2.css

```
@@ -5,19 +5,46 @@ body.mceContentBody{
+  --wpuf-l2-label: #6B738B;
+  --wpuf-l2-border: #33374D;
+  --wpuf-l2-accent: #00C14B;
+  --wpuf-l2-danger: #F20000;
+  --wpuf-l2-primary: #379EFF;
+  --wpuf-l2-help: #6F4A70;
+  --wpuf-l2-info-border: #D5AED9;
+  --wpuf-l2-info-bg: #F8F1FA;
+  --wpuf-l2-star-bg: #F8F1FA;
+  --wpuf-l2-star-inactive: #33374d8a;
+  --wpuf-l2-star-active: #EDB867;
   position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)